constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -495,3 +506,133 @@ func TestRegoRewriterErrorCases(t *testing.T) {
 		}
 	}
 }
+
+func TestRegoRewriterRewriteDataRef(t *testing.T) {
+	tests := []struct {
+		name    string
+		prefix  string
+		imports string
+		want    string
+	}{
+		{
+			name:    "no rewrite for ref is sub of externs",
+			prefix:  "foo.bar",
+			imports: "data.ext",
+			want:    "data.ext",
+		},
+		{
+			name:    "no rewrite for non data ref",
+			prefix:  "foo.bar",
+			imports: "invalid_data.name",
+			want:    "invalid_data.name",
+		},
+		{
+			name:    "add prefix for data ref",
+			prefix:  "foo.bar",
+			imports: "data.some_ref",
+			want:    "data.foo.bar.some_ref",
+		},
+		{
+			name:    "add prefix same as libs",
+			prefix:  "data.lib",
+			imports: "data.some_ref",
+			want:    "data.data.lib.some_ref",
+		},
+		{
+			name:    "add prefix same as libs",
+			prefix:  "data.lib",
+			imports: "data.lib.util",
+			want:    "data.data.lib.lib.util",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			r, err := New(NewPackagePrefixer(tt.prefix), []string{"data.lib"}, []string{"data.ext", "data.inventory"})
```

Comment:
Right now `[]string{"data.lib"}` and `[]string{"data.ext", "data.inventory"}` are functioning as magic values - I don't know what they mean in the context of the test. Assign these to variables with descriptive names, like `allowedPrefix` so it's clearer what their purpose is.

Reply:
Updated.